test/packages/other/long_integers/manifest.yml

```diff
@@ -0,0 +1,22 @@
+format_version: 1.0.0
+name: long_integers
+title: "Long Integer Tests"
+version: 0.0.1
+license: basic
+description: "These are tests of long integer round tripping."
```

Comment:
Nit. Maybe mention here in the comment that this refers to check that numbers don't lose precission during json decoding.

Reply:
I'm not sure that's helpful. The test should test for all failures to perfectly round-trip the long integer values. The intention here is to catch the low bit truncation, but it would also catch overflow regressions if they were to happen.